Unable to install man pages

#### The bug

I'm unable to install man pages.
I would like for example `man -man erlang` to work.

#### Software versions

A list of software versions where the bug is apparent, as detailed as possible:

* Operating System: I tried both on mac OS Sonoma 14.4.1 and Ubuntu 22.04
* Erlang/OTP: 26.2.4
* `kerl`: 4.1.0

#### How to replicate

This is on Ubuntu 22.04

1.
```
/bin/cat ~/.kerlrc

export KERL_BUILD_DOCS=yes
export KERL_DOC_TARGETS="man chunks"
export KERL_INSTALL_MANPAGES=yes
export KERL_DEFAULT_INSTALL_DIR="$HOME"/kerl
```

2.
```
kerl build 26.2.4

Extracting source code for normal build...
Building (normal) Erlang/OTP 26.2.4 (26.2.4); please wait...
Initializing (build) log file at /home/david/.kerl/builds/26.2.4/otp_build_26.2.4.log.
APPLICATIONS DISABLED (See: /home/david/.kerl/builds/26.2.4/otp_build_26.2.4.log)
 * odbc           : ODBC library - link check failed

APPLICATIONS INFORMATION (See: /home/david/.kerl/builds/26.2.4/otp_build_26.2.4.log)
 * wx             : No OpenGL headers found, wx will NOT be usable
 * No GLU headers found, wx will NOT be usable
 * wxWidgets was not compiled with --enable-webview or wxWebView developer package is not installed, wxWebView will NOT be available
 *         wxWidgets must be installed on your system.
 *         Please check that wx-config is in path, the directory
 *         where wxWidgets libraries are installed (returned by
 *         'wx-config --libs' or 'wx-config --static --libs' command)
 *         is in LD_LIBRARY_PATH or equivalent variable and
 *         wxWidgets version is 3.0.2 or above.

Building docs...
ERROR: building docs failed.
make[2]: Nothing to be done for 'docs'.
make[2]: Leaving directory '/home/david/.kerl/builds/26.2.4/otp_src_26.2.4/system/doc/programming_examples'
make[2]: Entering directory '/home/david/.kerl/builds/26.2.4/otp_src_26.2.4/system/doc/top'
sed -e 's;%VSN%;14.2.4;' \
    -e 's;%SYSTEM_VSN%;26;' \
    PR.template.src > PR.template
make[2]: Leaving directory '/home/david/.kerl/builds/26.2.4/otp_src_26.2.4/system/doc/top'
make[1]: Leaving directory '/home/david/.kerl/builds/26.2.4/otp_src_26.2.4/system/doc'
test -f /home/david/.kerl/builds/26.2.4/otp_src_26.2.4/make/otp_doc_built || echo "OTP doc built" > /home/david/.kerl/builds/26.2.4/otp_src_26.2.4/make/otp_doc_built
make: *** No rule to make target 'chunks'.  Stop.

Please see /home/david/.kerl/builds/26.2.4/otp_build_26.2.4.log for full details.
Auto cleaning all artifacts except the log file...
(use KERL_AUTOCLEAN=0 to keep build on failure, if desired)
Cleaning up compilation products for '26.2.4' under:
  - /home/david/.kerl/builds...
  - /home/david/.kerl/archives...
... done.
```
 
Log file `/home/david/.kerl/builds/26.2.4/otp_build_26.2.4.log` doesn't contain additional clues.

#### Expected behaviour

Build succeeds

#### Additional context

I get the same error `make: *** No rule to make target 'chunks'.  Stop.` on Mac OS.

I also tried with only
```
export KERL_INSTALL_MANPAGES=yes
export KERL_DEFAULT_INSTALL_DIR="$HOME"/kerl
```
in my `~/.kerlrc` file.
The build succeeds, however, this won't install the man pages:
```
erl -man erlang
No manual entry for erlang
No manual entry for --
```

--- a/kerl
+++ b/kerl
@@ -1196,11 +1196,11 @@ _do_build() {
     if [ -n "$KERL_BUILD_DOCS" ]; then
         notice "Building docs..."
         # shellcheck disable=SC2154  # KERL_DOC_TARGETS is referenced but not assigned
-        if ! log_build_cmd "make docs DOC_TARGETS=$KERL_DOC_TARGETS"; then
+        if ! log_build_cmd "make docs DOC_TARGETS=\"$KERL_DOC_TARGETS\""; then
             fail_do_build "building docs failed." "$2" "$1"
             return 1
         fi
-        if ! log_build_cmd "make release_docs DOC_TARGETS=$KERL_DOC_TARGETS RELEASE_ROOT=$KERL_BUILD_DIR/$2/release_$1"; then
+        if ! log_build_cmd "make release_docs DOC_TARGETS=\"$KERL_DOC_TARGETS\" RELEASE_ROOT=$KERL_BUILD_DIR/$2/release_$1"; then
             fail_do_build "release of docs failed." "$2" "$1"
             return 1
         fi
@@ -1722,7 +1722,7 @@ do_install() {
             error "couldn't cd into $ERL_TOP."
             exit_install
         fi
-        if ! log_install_cmd "ERL_TOP=$ERL_TOP make release_docs DOC_TARGETS=$KERL_DOC_TARGETS RELEASE_ROOT=$absdir"; then
+        if ! log_install_cmd "ERL_TOP=$ERL_TOP make release_docs DOC_TARGETS=\"$KERL_DOC_TARGETS\" RELEASE_ROOT=$absdir"; then
             show_install_logfile "couldn't install docs for Erlang/OTP $rel ($build_name) in $absdir"
             exit_install
         fi

